apps/etterlatte-saksbehandling-ui/client/src/components/behandling/revurderingsoversikt/Revurderingsoversikt.tsx

```diff
@@ -81,6 +82,9 @@ export const Revurderingsoversikt = (props: { behandling: IDetaljertBehandling }
         </BodyShort>
       </ContentHeader>
       <Innhold>
+        {behandling.revurderingsaarsak === Revurderingsaarsak.SOESKENJUSTERING ? (
```

Comment:
Bruk heller `&&` i stedet for ternary

Reply:
Lurer på hvor mange ganger vi skal ha den diskusjonen her 😆

Reply:
haha true